api/core/tools/provider/builtin/redis_tool/tools/set_string.yaml

```diff
@@ -0,0 +1,54 @@
+identity:
+  name: set_string
+  author: G81192
+  label:
+    en_US: set string
+    zh_Hans: 设置字符串值
+    pt_BR: set string
+description:
+  human:
+    en_US: A tool for redis set string
+    zh_Hans: redis设置字符串值
+    pt_BR: A tool for redis set string
+  llm: A tool for redis set string
+parameters:
+  - name: key
+    type: string
+    required: true
+    label:
+      en_US: key
+      zh_Hans: key
+      pt_BR: key
+    human_description:
+      en_US: key
+      zh_Hans: key
+      pt_BR: key
+    llm_description: key
+    form: llm
```

Comment:
```suggestion
    form: form
```
The value of `key` should not be decided by LLM, which is very dangerous.

Reply:
you're right